CONTRIBUTING.rst

```diff
@@ -0,0 +1,83 @@
+Contributing to emissions-api
+=============================
+
+As an open source project, emissions-api welcomes contributions of many forms.
+Examples of contributions include:
+
+- Code patches
+- Documentation improvements
+- Bug reports and patch reviews
```

Comment:
I like this introduction text as it invtes the reader get a part of this project.
I wuld like to propose two minor changes:
- Change many to any  `emissions-api welcomes contributions of any form`
- In the example of contributions, one important form is missing -> help developing new features (or is this meant by 'Code patches')